### ISSUE TITLE
build failure with Node 10.0.0

### ISSUE THREAD

**Issue by createthis** [abbr/deasync#97]
Not sure what's going on here:

```bash
> deasync@0.1.12 install /Users/jesse/Documents/CreateThis.com/Clients/LibertasFunding/node/node_modules/deasync
> node ./build.js

  CXX(target) Release/obj.target/deasync/src/deasync.o
In file included from ../src/deasync.cc:3:
In file included from ../../nan/nan.h:192:
../../nan/nan_maybe_43_inl.h:112:15: error: no member named 'ForceSet' in 'v8::Object'
  return obj->ForceSet(isolate->GetCurrentContext(), key, value, attribs);
         ~~~  ^
In file included from ../src/deasync.cc:3:
../../nan/nan.h:834:18: warning: 'MakeCallback' is deprecated: Use MakeCallback(..., async_context)
      [-Wdeprecated-declarations]
    return node::MakeCallback(
                 ^
/Users/jesse/.node-gyp/10.0.0/include/node/node.h:171:1: note: 'MakeCallback' has been explicitly marked
      deprecated here
NODE_DEPRECATED("Use MakeCallback(..., async_context)",
^
/Users/jesse/.node-gyp/10.0.0/include/node/node.h:88:20: note: expanded from macro 'NODE_DEPRECATED'
    __attribute__((deprecated(message))) declarator
                   ^
In file included from ../src/deasync.cc:3:
../../nan/nan.h:849:18: warning: 'MakeCallback' is deprecated: Use MakeCallback(..., async_context)
      [-Wdeprecated-declarations]
    return node::MakeCallback(
                 ^
/Users/jesse/.node-gyp/10.0.0/include/node/node.h:164:1: note: 'MakeCallback' has been explicitly marked
      deprecated here
NODE_DEPRECATED("Use MakeCallback(..., async_context)",
^
/Users/jesse/.node-gyp/10.0.0/include/node/node.h:88:20: note: expanded from macro 'NODE_DEPRECATED'
    __attribute__((deprecated(message))) declarator
                   ^
In file included from ../src/deasync.cc:3:
../../nan/nan.h:864:18: warning: 'MakeCallback' is deprecated: Use MakeCallback(..., async_context)
      [-Wdeprecated-declarations]
    return node::MakeCallback(
                 ^
/Users/jesse/.node-gyp/10.0.0/include/node/node.h:157:1: note: 'MakeCallback' has been explicitly marked
      deprecated here
NODE_DEPRECATED("Use MakeCallback(..., async_context)",
^
/Users/jesse/.node-gyp/10.0.0/include/node/node.h:88:20: note: expanded from macro 'NODE_DEPRECATED'
    __attribute__((deprecated(message))) declarator
                   ^
In file included from ../src/deasync.cc:3:
../../nan/nan.h:1473:31: warning: 'MakeCallback' is deprecated: Use MakeCallback(..., async_context)
      [-Wdeprecated-declarations]
    return scope.Escape(node::MakeCallback(
                              ^
/Users/jesse/.node-gyp/10.0.0/include/node/node.h:171:1: note: 'MakeCallback' has been explicitly marked
      deprecated here
NODE_DEPRECATED("Use MakeCallback(..., async_context)",
^
/Users/jesse/.node-gyp/10.0.0/include/node/node.h:88:20: note: expanded from macro 'NODE_DEPRECATED'
    __attribute__((deprecated(message))) declarator
                   ^
4 warnings and 1 error generated.
make: *** [Release/obj.target/deasync/src/deasync.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:258:23)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:225:12)
gyp ERR! System Darwin 17.5.0
gyp ERR! command "/usr/local/Cellar/node/10.0.0/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/jesse/Documents/CreateThis.com/Clients/LibertasFunding/node/node_modules/deasync
gyp ERR! node -v v10.0.0
gyp ERR! node-gyp -v v3.6.2
gyp ERR! not ok 
Build failed
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! deasync@0.1.12 install: `node ./build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the deasync@0.1.12 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jesse/.npm/_logs/2018-05-08T17_24_59_361Z-debug.log
```

Any ideas?

**Comment by nwhitmont** [comment]
same here

**Comment by abbr** [comment]
Should be fixed by v0.1.13
